data.sql

```diff
@@ -1,5 +1,10 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered,weight_kg, species) VALUES (1,'Augmon', '02/03/2020', 0, true, 10.23);
+INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered,weight_kg, species) VALUES (2,'Gabumon', '10/10/2018', 2, true, 8);
+INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered,weight_kg, species) VALUES (3,'Pikachu', '06/07/2021', 1, false, 15.04);
+INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered,weight_kg, species) VALUES (4,'Devimon', '05/12/2017', 5, true, 11);
+INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered,weight_kg, species) VALUES (1,'Augmon', '02/03/2020', 0, true, 10.23);
+INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered,weight_kg, species) VALUES (2,'Gabumon', '10/10/2018', 2, true, 8);
+INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered,weight_kg, species) VALUES (3,'Pikachu', '06/07/2021', 1, false, 15.04);
+INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered,weight_kg, species) VALUES (4,'Devimon', '05/12/2017', 5, true, 11);
```

Comment:
- [ ] Kindly change the DATE_FORMAT, PostgreSQL uses the yyyy-mm-dd format for storing and inserting date values you should YYYY-MM-DD for more documentation visit [PostgreSQL – Date Data Type](https://www.geeksforgeeks.org/postgresql-date-data-type/#:~:text=PostgreSQL%20uses%20the%20yyyy%2Dmm,CURRENT_DATE%20after%20the%20DEFAULT%20keyword.&text=Let's%20look%20into%20some%20examples%20for%20better%20understanding.)